source/tutorials/fmt-cdiff.rst

```diff
@@ -0,0 +1,478 @@
+FMT for recurrent Clostridium difficile infection Tutorial
+==========================================================
+
+.. note:: This guide assumes you have installed QIIME 2 using one of the procedures in the :doc:`install documents <../install/index>`.
+
+In this tutorial you'll use QIIME 2 to perform an analysis
+In this tutorial you’ll use QIIME 2 to perform an analysis of fecal human microbiome samples looking at short- and long-term changes in patients with multiply recurrent Clostridium difficile infection that were refractory to antibiotic therapy and treated using fecal microbiota transplantation. A study based on these samples was originally published in [Weingarden et al. (2015)](https://www.ncbi.nlm.nih.gov/pubmed/25825673) and it has been used in [animations](https://www.youtube.com/watch?v=-FFDqhM4pks) and [meta-analyses](https://github.com/knightlab-analyses/qiita-paper). The data used in this tutorial were sequenced on an Illumina MiSeq using the [Earth Microbiome Project](http://earthmicrobiome.org/) hypervariable region 4 (V4) 16S rRNA sequencing protocol.
+
+Before beginning this tutorial, create a new directory and change to that directory.
+
+.. command-block::
+   :no-exec:
+
+   mkdir qiime2-fmt-cdi-tutorial
+   cd qiime2-fmt-cdi-tutorial
+
+Sample metadata
+---------------
+
+Before starting the analysis, explore the sample metadata to familiarize yourself with the samples used in this study. [ToDo: add metadata to Google spreadsheets?]. The following command will download the sample metadata as tab-separated text and save it in the file  `sample-metadata.tsv`. This  `sample-metadata.tsv` file is used throughout the rest of the tutorial.
```

Comment:
having a google spreadsheet could be useful for data exploration, but not necessary

Reply:
New URL:

https://data.qiime2.org/2018.11/tutorials/cdiff-fmt/sample_metadata

Reply:
> Will add, do we have an "official" account for this?

Yep --- I shared the file with you at your gmail account earlier this morning.